mantidproject/mantid
# Remove unused MATLAB bindings

Description of work.

The MATLAB bindings are unused, untested and most likely don't work. It was decided that these should be removed.

**To test:**

Check that the builds still pass.



There is no GitHub issue associated with this pull request.

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
